src/controllers/update/update.manage.trusts.tell.us.about.the.former.bo.controller.ts

```diff
@@ -101,3 +106,11 @@ const getBackLink = (formerBosReviewed) => {
     return UPDATE_MANAGE_TRUSTS_REVIEW_FORMER_BO_URL;
   }
 };
+
+const checkErrors = async (req: Request): Promise<ValidationError[]> => {
```

Comment:
It feels like this function if checking _for_ errors. It's really just validating the trust start date and cease date (I think) so how about calling it `validateFilingPeriodDates()`? The returned result is then 'checked'.

Reply:
This name is the same in all controllers because the job is similar: to get validation errors that are not in errorList.
The name is as common as can be errorList = validationResult(req)

Reply:
I'd rather a specific name - it's not having to implement a common interface function or anything - but regardless, renaming to `getValidationErrors` would be better I think because that matches what you said in your comment. The actual checking of the errors happens on line 56 (I think) when you check the length of the array.

Reply:
I agree with the name getValidationErrors

Reply:
name changed to getValidationErrors

Reply:
👍